app/views/onboarding/consent/_statement.html.erb

```diff
@@ -1,65 +1,24 @@
-<li>Names and contact information for people in your school.</li>
-</ul>
+<h4><%= t('consent.section_3_title') %></h4>
 
-<p>
-  The means of collecting this data will be specific to your school depending on your energy supplier and what type of meters you have installed. We will handle setting this on your behalf but may require additional information from you.
-</p>
+<%= t('consent.section_3_content_html', privacy_policy_link: link_to('privacy policy', privacy_and_cookie_policy_path, target: "_blank")) %>
 
-<p>More information about how we store and use your data is covered in our <%= link_to 'privacy policy', privacy_and_cookie_policy_path, target: "_blank" %>.</p>
+<h4><%= t('consent.section_4_title') %></h4>
 
-<p>We also use public information about your school, including its address to verify which meters you have installed.</p>
+<%= t('consent.section_4_content_html', terms_and_conditions_link: link_to('Terms and Conditions', terms_and_conditions_path, target: "_blank")) %>
 
+<h4><%= t('consent.section_5_title') %></h4>
 
-<h4>What we will do with your data</h4>
-
-<p>As described in our <%= link_to 'privacy policy', privacy_and_cookie_policy_path, target: "_blank" %>, we will use this data to:</p>
-
-<ul>
-<li>Analyse and present your energy consumption and generation data on our website</li>
-<li>Email or text you alerts highlighting changes in energy use, or long term energy consumption problems</li>
-<li>Email you our regular newsletter</li>
-<li>Support you with saving energy at your school</li>
-</ul>
-
-<p>Energy Sparks will collect and use consumption, generation and export data for as long as your school uses our service.</p>
-<p>Some of your data, including information on energy usage will be publicly displayed on our website. Other data, such as accurate tariff information is only presented to authorised school users. Publicly accessible cost estimates are calculated using average cost data.</p>
-<p>Independent schools may opt-in to make all data private, for an additional fee.</p>
-
-<h4>Withdrawing consent</h4>
-
-<p>Your school has the right to withdraw consent at any time.</p>
-
-<p>As described in our <%= link_to 'Terms and Conditions', terms_and_conditions_path, target: "_blank" %> you can do this by emailing hello@energysparks.uk. Energy Sparks
-  will then remove your ongoing and historical energy data from the Energy Sparks website within one month.
-</p>
-
-<h4>Your grant of consent</h4>
-
-<p>To complete the setup of your school, please read the following consent statement and provide your name and job title.</p>
-
-<p>You must also agree to our <%= link_to 'Terms and Conditions', terms_and_conditions_path, target: "_blank" %> which set out the full agreement between your school and Energy Sparks.</p>
+<%= t('consent.section_5_content_html', terms_and_conditions_link: link_to('Terms and Conditions', terms_and_conditions_path, target: "_blank")) %>
```

Comment:
does 'Terms and Conditions' in the link_to need a translation?

Reply:
ah of course, good spot